### PR TITLE
Price Signal Dispatch

### PR DESCRIPTION
**Pull request by brtietz**
Merge price signal dispatch algorithm into develop. Should be merged after SSC PR: https://github.com/NREL/ssc/pull/451 .

Changes:
- Split dispatch page from main battery storage page (@cpaulgilman, note: a new help page will be needed)
- Rename "battery cycle costs" to "cycle degradation penalty" in the GUI
- Make the cycle degradation penalty an array so the user can specify a schedule
- Add new dispatch option

Let me know if additional changes are needed to help pages, version upgrade script, or similar.

### CODE REVIEW

**Review by cpaulgilman — APPROVED** [review]
This looks good to me, Brian. I agree with Steve that we should merge sooner than later. I pushed an update to startup.lk to point to new Help topics that I will commit later -- I don't want to clutter up this pull request with a bunch of new HTML and PNG Help files.

File: deploy/runtime/startup.lk [hunk 1]
```diff
@@ -115,16 +115,19 @@ function setup_battery_pages_behind_the_meter()
...
-			   'Battery Dispatch Common',
-			   'Battery Dispatch Behind the Meter',
-			   'Battery Dispatch Manual',
                {'name'='Battery Lifetime', 'caption'='Battery Lifetime', 'collapsible'=true, 'collapsible_var'='battery_lifetime_shown'},
 			   {'name'='Battery Replacements', 'caption'='Battery Replacements', 'collapsible'=true, 'collapsible_var'='battery_replacements_shown'},
 			   {'name'='Battery Voltage', 'caption'='Battery Voltage', 'collapsible'=true, 'collapsible_var'='battery_voltage_shown'},
 			   {'name'='Battery Ancillary Losses', 'caption'='Battery Losses', 'collapsible'=true, 'collapsible_var'='battery_losses_shown'},
 			   {'name'='Battery Thermal', 'caption'='Battery Thermal', 'collapsible'=true, 'collapsible_var'='battery_thermal_shown'},
 			  ]],
 			  { 'sidebar'='Battery Storage', 'help'='battery_storage_btm' } );
```
Comment (dguittet) [inline deploy/runtime/startup.lk:124]: @cpaulgilman @brtietz Can I suggest calling this "Battery Cell and System" to align with the battery's SSC group names?

File: deploy/runtime/ui/Battery Dispatch Behind the Meter.txt [hunk 2]
```diff
@@ -177,6 +207,169 @@ For each simulation time step enter a rate in kW: less than zero to charge the b
...
+Height
+3
+69
+Tool Tip
+5
+0
+Caption
+5
+305
+The dispatch options will cycle the battery if anticipated revenue exceeds the cycle degradation penalty. A higher penalty will decrease cycling, a lower penalty will increase cycling and may lead to additional battery replacements. See the "Battery Storage - Battery Lifetime" page for degradation modes.
```
Comment (dguittet) [inline deploy/runtime/ui/Battery Dispatch Behind the Meter.txt:312]: name also here

**Review by dguittet — CHANGES_REQUESTED** [review]
Looks great! I think this is a welcome change.

Thoughts on naming the UI page?

**Pull request merged**